### PR TITLE
Don't use a submodule for secrets

### PR DESCRIPTION
**Pull request by mbrandonw**
We're having lots of problems with using submodules for the secrets. Im just gonna copy what we are doing in iOS and clone it straight up...

### CODE REVIEW

**Comment by stephencelis** [comment]
@mbrandonw Actually, looks like CI is failing on secrets-related things.

File: Makefile [hunk 1]
```diff
@@ -11,26 +11,16 @@ submodules:
...
-		|| cp config/koala_endpoint.xml.example app/src/main/res/values/koala_endpoint.xml
-
-	# Copy over web client
-	cp -rf vendor/native-secrets/android/WebViewJavascriptInterface.java app/src/main/java/com/kickstarter/libs/WebViewJavascriptInterface.java \
-		|| cp -rf config/WebViewJavascriptInterface.java app/src/main/java/com/kickstarter/libs/WebViewJavascriptInterface.java
-
-	cp -rf vendor/native-secrets/android/KSWebViewClient.java app/src/main/java/com/kickstarter/services/KSWebViewClient.java \
-		|| cp -rf config/KSWebViewClient.java app/src/main/java/com/kickstarter/services/KSWebViewClient.java
+	-rm -rf vendor/native-secrets
+	-git clone https://github.com/kickstarter/native-secrets1 vendor/native-secrets
```
Comment (luoser) [inline Makefile]: should be `/native-secrets` not `/native-secrets1`

Comment (stephencelis) [inline Makefile]: 😅

Comment (mbrandonw) [inline Makefile]: lol. `1` lil thing indeed

**Pull request merged**